### ISSUE TITLE
Settings types in options to cities not working

### ISSUE THREAD

**Issue by quachsimon** [kenny-hibino/react-places-autocomplete#136]
I am trying to set the `types` in the options parameters to limit the search to just cities.

    const options = {
      location: new google.maps.LatLng(85, -180),
      radius: 5000,
      types: ['cities']
    }

When I do this, I receive invalid requests from the `onError` callback.

I am following the [docs](https://developers.google.com/maps/documentation/javascript/3.28/reference#AutocompletionRequest) and those are one of the options.

What am I missing?

**Comment by cicer1** [comment]
Same problem for me.

**Comment by quachsimon** [comment]
@cicer1  It wasn't obvious but, I had to wrap it around parentheses exactly like the docs `(cities)` so you would have to put it like so:  `types: ['(cities)']`

**Comment by rfreethy** [comment]
I am having this issue trying to limit the types to address.

`
                  searchOptions={{
                    types: ["address"],
                    country: ["ca", "us"]
                  }}
`

Is returning INVALID_REQUESTS

**Comment by brendonlamb** [comment]
> @cicer1 It wasn't obvious but, I had to wrap it around parentheses exactly like the docs `(cities)` so you would have to put it like so: `types: ['(cities)']`

Serious thanks for this one, great catch, wish google was little more clear about the syntax... 

For others looking for the actual reference on this:

https://developers.google.com/places/supported_types#table3

**Comment by itsmhuang** [comment]
This is still broken or broken _again_. Setting `types: ['(cities)']` doesn't limit the search results to cities, the search results still returns all types.

**Comment by cmacdonnacha** [comment]
@itsmhuang , are you still seeing this issue? I am.

**Comment by m-jabrane** [comment]
I don't know why! but if you create a variable with customized parameters and then give that variable to the `searchOptions` parameter it's not gonna work. I had to give these customized parameters directly in the component like:
```                     <PlacesAutocomplete
                              onChange={this.Change}
                              onSelect={this.handleSelect}
                              searchOptions={{
                                componentRestrictions: {
                                  country: ['us'],
                                },
                                types: ['(cities)']
                              }}
                          >
                          ```

**Comment by m-jabrane** [comment]
I don't know why! but if you create a variable with customized parameters and then give that variable to the `searchOptions` parameter it's not gonna work. I had to give these customized parameters directly in the component like: 
```
<PlacesAutocomplete
  onChange={this.Change}
  onSelect={this.handleSelect}
  searchOptions={{
    componentRestrictions: {
    country: ['us'],
    },
    types: ['(cities)']
  }}
>
```

**Comment by brendonlamb** [comment]
Yeah this is definitely not working well again :( and @m-jabrane 's solution isn't working for me either, using that exact code my autocomplete isn't filtering by cities only OR by country (US):
![image](https://user-images.githubusercontent.com/1816511/69904390-18b6f900-1374-11ea-805e-068339adb2cd.png)

However, removing the "types" property all together does appear to make the country filter work... :/

We're considering just allowing all "places" through anyway... cities only has just been a headache :/ especially in rural areas, and even Hawaii:

![image](https://user-images.githubusercontent.com/1816511/69904803-2c189300-1379-11ea-9207-fdf025cd1b2c.png)

**Comment by nescroft** [comment]
i was able to get it working for REACT NATIVE with query={{ types: '(cities)',}} (no array) using https://github.com/FaridSafi/react-native-google-places-autocomplete
